pkgs/tools/security/cryptomator/default.nix

```diff
@@ -102,7 +102,7 @@ in stdenv.mkDerivation rec {
     autoPatchelfHook
     maven
     makeWrapper
-    (wrapGAppsHook.override { makeBinaryWrapper = makeWrapper; })
+    (wrapGAppsHook.override { inherit makeWrapper; })
```

Comment:
```suggestion
    wrapGAppsHookp
```
Just replace makeWrapper with makeShellWrapper.

Reply:
Indeed, that seems better.